src/Illuminate/Auth/AuthenticationException.php

```diff
@@ -3,16 +3,26 @@
 namespace Illuminate\Auth;
 
 use Exception;
+use Illuminate\Support\Collection;
 
 class AuthenticationException extends Exception
 {
+    protected $guards = [];
+
     /**
      * Create a new authentication exception.
      *
-     * @param string  $message
+     * @param string|array $guards
+     * @param string $message
      */
-    public function __construct($message = 'Unauthenticated.')
+    public function __construct($guards = [], $message = 'Unauthenticated.')
```

Comment:
I'm working on 5.3.21 and I'm trying to use @leeovery example but there is 2 problem is still exist:
`$this->guard()` is always empty and I get error using `->contains('admin')`
`Call to a member function contains() on array`. The error is because it's not possible to call function on an array but the first issue is not recognizable for me.